src/TLuaInterpreter.cpp

```diff
@@ -309,6 +309,11 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             pE.mArgumentList.append( QString(lua_tostring( L, i )) );
             pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
         }
+        else if( lua_isboolean( L, i ) )
+        {
+            pE.mArgumentList.append( QString::number(lua_toboolean( L, i )) );
```

Comment:
If I'm understanding you correctly, I don't see any reason that would be particularly objectionable, other than requiring slightly more verbose code. I also personally don't see the argument in favor, either, but I haven't worked with the Qt IDE, so maybe I'm missing something.